Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/Search.hs

```diff
@@ -91,7 +91,8 @@ buildSearchReq merchantId subscriber req = do
         device = Nothing,
         routePoints = buildRoutePoints =<< intent.fulfillment.tags, --------TODO------Take proper input---------
         customerLanguage = customerLanguage,
-        disabilityTag = disabilityTag
+        disabilityTag = disabilityTag,
+        cancellationDues = buildCancellationDues =<< intent.fulfillment.customer
```

Comment:
Why cancellation dues are coming from BAP, they should be BPP for the particular rider, right? Not at BAP.

Reply:
I asked the same to @sidsethu but we discussed that since it is related to BAP so it should be stored on BAP side.